nin/dasBoot/LayerManager.js

```diff
@@ -49,9 +49,20 @@ LayerManager.prototype.loadLayer = function(layer) {
 LayerManager.prototype.update = function(frame) {
   this.updateActiveLayersList(frame);
   for(var i = 0; i < this.activeLayers.length; i++) {
-    var relativeFrame = frame - this.activeLayers[i].startFrame;
-    this.activeLayers[i].instance && this.activeLayers[i].instance.update(
-        frame, relativeFrame);
+    var layer = this.activeLayers[i],
+        li = layer.instance,
+        relativeFrame = frame - layer.startFrame;
+
+    if (li) {
+      li.update(frame, relativeFrame);
+
+      if (li.cameraController &&
+          li.cameraController.position &&
+          li.cameraController.position.visualizer &&
+          frame >= layer.startFrame && frame <= layer.endFrame) {
+        li.cameraController.position.visualizer.update(relativeFrame);
```

Comment:
Agree. Perhaps a LayerManager variable set to true if visualizations are to be shown?

This part of the code is also a bit brittle, as skipping in the demo with visualizatinos enabled sometimes breaks the update loop here, apparently update is called before layers are removed or something on skip.